.github/workflows/pre-merge.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: pre-merge
+
+on:
+  push:
+    branches:
+    - 'user/**'
+    - 'feature/**'
+    - 'improvement/**'
+    - 'bugfix/**'
+    - 'w/**'
+    - 'q/**'
+    - 'dependabot/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      redis:
+        # Docker Hub image
+        image: redis
+        # Set health checks to wait until redis has started
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps port 6379 on service container to the host
+          - 6379:6379
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '10'
+        cache: 'yarn'
+    - name: install dependencies
+      run: yarn install --frozen-lockfile
+    - name: run lint yml
+      run: yarn run --silent lint_yml
+    - name: run lint
+      run: yarn run --silent lint -- --max-warnings 0
+    - name: run lint_md
+      run: yarn run --silent lint_md
+    - name: run test
+      run: yarn run --silent test
+    - name: run ft_test
+      run: yarn run ft_test
+    - name: run executables tests
+      run: yarn install && yarn test
```

Comment:
this looks duplicated unless there is some difference between `yarn install` and `yarn install --frozen-lockfile` which there shouldn't be.

Reply:
wow, I've copied paste what we had in eve, this means that we have been running duplicated install since forever :sweat_smile:

Reply:
actually, no I don't they are duplicated as the yarn install is running in a different directory with a different `package.json` I believe we should leave it this way.

Reply:
oh yea I see the working-directory thing now